src/content/learning-resources-index/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Flutter learning resources
+description: A catalog of Flutter sample applications, codelabs, and tutorials.
+short-title: Learning resources
+show_breadcrumbs: false
+layout: wide
+toc: false
+js: [ { url: '/assets/js/learning-resources-index.js', defer: true } ]
+---
+
+:::secondary
+This page lists all of our additional learning resources:
+* Cookbook recipes that demonstrate how to solve common problems with Flutter.
+* Guided Codelabs and tutorials that walk you through building features and applications.
+* Working sample applications that show how to use Flutter.
+* Feature-rich demo applications that show how larger applications are built.
+:::
+
+{% assign resources = learning-resources-index.codelabs | concat: learning-resources-index.cookbook | concat: learning-resources-index.demos | concat: learning-resources-index.quickstarts_flutter | concat: learning-resources-index.quickstarts_dart | shuffle -%}
```

Comment:
This shuffle will only be updated with each build/deploy, do you think that's sufficient for analytics purpose?